src/everwillow/bounds.py

```
@@ -0,0 +1,292 @@
+import typing as tp
+
+import jax.numpy as jnp
+from jaxtyping import Array, Float
+
+import everwillow.statelib as sl
+
+# Type alias for bounds specification
+BoundSpec = tuple[float | None, float | None] | None
+
+
+def _logit(x: Float[Array, ""]) -> Float[Array, ""]:
```

Ruff (0.14.0)

Syntax error in forward annotation: Expected an expression

(F722)